tests/test_MessageMesh.py

```diff
@@ -18,6 +18,9 @@
 x_size = int(sys.argv[1])
 y_size = int(sys.argv[2])
 
+sst.setStatisticOutput("sst.statOutputCSV")
```

Comment:
Using the default name for stats output file in the test could cause random failures when running testsuites in parallel.  However, I think this is a problem in other testsuites as well and we don't currently run the tests in parallel so we can reevaluate this one when we look at the others.

Reply:
I can override it with a specific name if need be

Reply:
Let's leave it as is for now.  This just got me thinking about parallel testsuites running and I found errors in other testsuites as well.  We'll tackle them all together.